features/support/env.rb

```diff
@@ -101,4 +102,12 @@
 
 Before do |_scenario|
   load Rails.root.join('db/seeds.rb')
+  Warden.test_mode!
 end
+
+After do |_scenario|
+  Warden.test_reset!
+end
```

Comment:
Not a blocker: Would [an around block](https://relishapp.com/cucumber/cucumber/docs/writing-support-code/around-hooks) be good here, as it would have both the set and reset actions within the same block:
```ruby
Around do |_scenario, block|
  Warden.test_mode!
  block.call
  Warden.test_reset!
end
```